governance/sig-governance.md

```diff
@@ -0,0 +1,90 @@
+# SIG Governance
+
+Special Interest Groups (SIGs) are persistent open groups that focus on a module of TiDB. SIGs must have open and transparent proceedings. The purpose of a SIG is to own and develop a module of TiDB. This document details on the governance guidelines of TiDB SIGs. It will be updated as needed to meet the current needs of the TiDB Community.
```

Comment:
```suggestion
Special Interest Groups (SIGs) are persistent open groups that focus on some modules of TiDB. SIGs must have open and transparent proceedings. The purpose of a SIG is to own and develop a module of TiDB. This document details on the governance guidelines of TiDB SIGs. It will be updated as needed to meet the current needs of the TiDB Community.
```

Reply:
I think the definition of sig in K8s community is more proper:

```
The Kubernetes project is organized primarily into Special Interest Groups, or SIGs. Each SIG is comprised of members from multiple companies and organizations, with a common purpose of advancing the project with respect to a specific topic, such as Networking or Documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. Every identifiable subpart of the project (e.g., github org, repository, subdirectory, API, test, issue, PR) is intended to be owned by some SIG.

Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:

Vertical: Network, Storage, Node, Scheduling
Horizontal: Scalability, Architecture
Project: Testing, Release, Docs, PM, Contributor Experience
```